CHANGELOG.rst

```diff
@@ -24,7 +24,7 @@ This project adheres to `Semantic Versioning <http://semver.org/>`_.
 * Replace built-in email support with django-templated-mail
 * Refactor test configuration for better performance and organization
 * Refactor RootView to have better support for modular URLs
-* Update URLs to be slightly more RESTful.
+* Update URLs to be slightly more RESTful. **NOTE**: For Token-based and JWT authentication, this means that 'djoser.urls' should be included in the URLconf as well as either 'djoser.urls.authtoken' or 'djoser.urls.jwt'
```

Comment:
Just to make it more readable and visible - could we perhaps move it to the top and slightly reformat it to something alike:

```
* **Breaking**: For Token-based and JWT authentication, this means that ``djoser.urls`` should be included in the URLconf as well as either ``djoser.urls.authtoken`` or ``djoser.urls.jwt``
```

Besides that, it's awesome and I'd love to merge it.